js/modules/modules.go

```diff
@@ -1,3 +1,4 @@
+// Package modules defines interfaces to be implemented by js modules to be imported by k6.
```

Comment:
Maybe something like this?
```suggestion
// Package modules defines interfaces used for developing k6's JavaScript modules.
```